docs/guides/run/health-checks.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Healthchecks
+description: description
+icon: heart-pulse
+---
+
+## Configuration
+
+Below you can find an example with a toml configuration file for a service with healthcheck enabled.
+
+```toml
+name = 'mysvc'
+
+[image]
+image = 'registry.eu-central-1.nhost.run/myimage:version'
+
+[[environment]]
+name = 'MY_ENV2'
+value = 'myvalue2'
+
+[[ports]]
+port = 3000
+type = 'http'
+publish = true
+
+[resources]
+replicas = 1
+
+[resources.compute]
+cpu = 62
+memory = 128
+
+[[resources.storage]]
+name = 'storage1'
+capacity = 1
+path = '/var/lib/storage1'
+
+[healthCheck]
+port = 3000
+initialDelaySeconds = 30
+probePeriodSeconds = 10
+```
+
+## Healthcheck behaviour
+
+The healthcheck endpoint must be available at `/healthz` on the port specified in `healthCheck.port`. If your container image exposes multiple ports only one of them can be used for healthchecks.
+
+The endpoint must return `200` status code within `5` seconds for the service to be considered healthy. If the service fails to respond correctly `3` times in a row it will be considered not healthy and will be restarted with an exponential backoff delay (10s, 20s, 40s, …), that is capped at five minutes.
+
+With `healthCheck.initialDelaySeconds` you can specify the number of seconds to wait before sending the first healthcheck request, this is useful if the service needs some time to start up. The default value is `30` seconds.
+
+With `healthCheck.probePeriodSeconds` you can specify the number of seconds to wait between healthcheck requests. The default value is `10` seconds.
```

Comment:
is the default value 10s? It's a bit low, 60s would probably be better for a default.

Reply:
Yeah I think so too, will fix that on BE repo later then.